cli/src/commands/federated-graph/commands/create.ts

```diff
@@ -20,12 +23,19 @@ export default (opts: BaseCommandOptions) => {
     '--label-matcher [labels...]',
     'The label matcher is used to select the subgraphs to federate. The labels are passed in the format <key>=<value> <key>=<value>. They are separated by spaces and grouped using comma. Example: --label-matcher team=A,team=B env=prod',
   );
+  createFederatedGraph.option('--readme <path-to-readme>', 'The markdown file which describes the federated graph.');
   createFederatedGraph.action(async (name, options) => {
+    let readmeFile;
+    if (options.readme) {
+      readmeFile = resolve(process.cwd(), options.readme);
```

Comment:
Please check if the file exists, if not throw an error.

Reply:
Done